cmd/container-suseconnect/main.go

```diff
@@ -33,7 +33,7 @@ func actionWrapper(action func(*cli.Context) error) func(*cli.Context) error {
 		if err := action(ctx); err != nil {
 			switch err.(type) {
 			case *cs.SuseConnectError:
-				if err.(*cs.SuseConnectError).ErrorCode == cs.GetCredentialsError {
+				if err.(*cs.SuseConnectError).Code == cs.GetCredentialsError {
```

Comment:
If the plan is to use `fmt.Errorf` you can use `errors.Is` (or `errors.As`) if you use `%w`.

Reply:
This code is going away with #93, this PR needs a rebase once that is merged.